.github/workflows/scripts-lint.yml

```diff
@@ -1,32 +0,0 @@
-name: Scripts
-
-on:
-  pull_request:
-    branches:
-      - main
-  push:
-    branches:
-      - main
-
-jobs:
-  shellcheck:
-    runs-on: ubuntu-latest
-    steps:
-    - name: Checkout
-      uses: actions/checkout@v2
-    - name: Run shellcheck
-      uses: ludeeus/action-shellcheck@1.1.0
-      env:
-        SHELLCHECK_OPTS: --external-sources
```

Comment:
We should keep this check - Forest codebase not only relies on shell scripts for the infrastructure code but also for a check like here https://github.com/ChainSafe/forest/blob/main/scripts/add_license.sh. It doesn't cost much to run shellcheck, few seconds at most which is run in parallel anyway.

Reply:
Rerturned